spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb

```diff
@@ -31,11 +31,9 @@ def initialize
 ENV['BEAKER_fips'] = 'yes'
 ENV['FIXTURES_YML'] = alt_fixtures
 
-Bundler.with_clean_env{
-  ENV['FIXTURES_YML'] = alt_fixtures
+ENV['FIXTURES_YML'] = alt_fixtures
```

Comment:
With the removal of the `Bundler.with_clean_env` block, this now duplicates the previous line.

Reply:
Hrm...propagation is probably a better idea, didn't think of that.